dev-python/python-gammu/Manifest

```diff
@@ -1 +1,2 @@
+DIST python-gammu-2.11.tar.gz 135567 BLAKE2B 133172cfd0715ff6b5bd47b93e5d93d52af3e608459ba1158bf008da9b5f24f6d155c04334dea7ff2cacdea678f00d986838a139290d5f8ac874bb9702fcd5e6 SHA512 cdf3a54fc538fecdbe3dae63f0534094ce94a289998f9b670a0b3e12a3c571eb46ee86eae67a7a8d2428698850046d5c75569753bf77551cd395d315e6f9b70d
```

Comment:
Body part of this commit message doesn't add anything compared to the summary line, so drop it.

Reply:
fixed